### PR TITLE
🎉 PEP8 styling guidelines

### PR DESCRIPTION
**Pull request by Mrinank-Bhowmick**
# Added PEP8 Checker 🥇 
###  I am adding a PEP8 checker to the github repo using GIthub Actions to prevent non-PEP8 compliant code from passing through.
_**Hope this will make the work easier for the future contributors.**_

### Changes proposed in this pull request
I have added the workflow in `.github/workflows` folder.
I have already made the required changes in PEP8 where there is required. 👍 

A successful run can be seen here for example : [https://github.com/MrinankBhowmick/Nettacker/pull/3/checks](https://github.com/MrinankBhowmick/Nettacker/pull/3/checks) , here I tried the workflow from one branch to another while making a pull request.

* [Official documentation of `Black`](https://black.readthedocs.io/en/stable/integrations/github_actions.html)
* [Lint Github action](https://github.com/marketplace/actions/lint-action#python-example-flake8-and-black) which will auto fix the non-PEP8 compliant code.

### Reviewers

@Ali-Razmjoo
@securestep9

#### Checklist
- [x] I have followed the [Contributor Guidelines](https://github.com/OWASP/Nettacker/wiki/Developers#contribution-guidelines).
- [x] The code has been thoroughly tested in my local development environment with flake8 and pylint.
- [x] The code is Python 3 compatible.
- [x] The code follows the PEP8 styling guidelines with 4 spaces indentation.
- [x] This Pull Request relates to only one issue or only one feature
- [x] I have referenced the corresponding issue number in my commit message
- [x] I have added the relevant documentation.
- [x] My branch is up-to-date with the Upstream master branch.

#### Your development environment
- OS: `Docker: Debian GNU/Linux 11 (bullseye)`
- OS Version: `11`
- Python Version: `3.9`

### CODE REVIEW

**Comment by MrinankBhowmick** [comment]
Written clean code which is in roadmap of this project , https://github.com/OWASP/Nettacker/projects/4#card-69507020 . 
Thanks

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        with:
+          python-version: 3.8
+
+      - name: Install Python dependencies
+        run: pip install black
+
+      - name: Run linters
+        uses: wearerequired/lint-action@v1
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
```
Comment (Ali-Razmjoo) [inline .github/workflows/lint.yml:25]: What `secrets.GITHUB_TOKEN` refers to?

**Review by Ali-Razmjoo — CHANGES_REQUESTED** [review]
could you please merge the master to this branch and fix the conflicts?

File: .github/workflows/lint.yml (hunk 1 above)
Comment (Mrinank-Bhowmick) [reply .github/workflows/lint.yml:25]: I guess its not in need for now

**Comment by Mrinank-Bhowmick** [comment]
> could you please merge the master to this branch and fix the conflicts?

I am making a fresh new pull request.

**Pull request closed without merging**